src/filesystem.rs

```diff
@@ -41,6 +59,11 @@ pub fn default_config_pathbuf() -> Result<PathBuf> {
     let mut pathbuf = PathBuf::from(base_dirs.config_dir());
     pathbuf.push("navi");
     pathbuf.push("config.yaml");
+    if !pathbuf.exists() {
+        if let Some(path) = compiled_default_path(option_env!("NAVI_CONFIG_FILE")) {
```

Comment:
Instead of creating a new env var, could we please reuse https://github.com/denisidoro/navi/blob/8725654a746fa8c999fbd5437559fdfcf1f7fb53/src/env_var.rs#L19?